test/utils/ReceiptDataHelperSpec.scala

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import java.time.LocalDate
+
+import base.BaseSpec
+import connectors.VatSubscriptionConnector
+import models.auth.User
+import models.errors.BadRequestError
+import models.nrs._
+import models.{CustomerDetails, SubmitVatReturnModel}
+import play.api.mvc.{AnyContentAsEmpty, Cookie}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ReceiptDataHelperSpec extends BaseSpec {
+
+  val mockConnector: VatSubscriptionConnector = mock[VatSubscriptionConnector]
+
+  val service = new ReceiptDataHelper(mockConnector, messagesApi)
+
+  val dateToUse: LocalDate = LocalDate.now()
+
+  def createReturnModel(flatRateScheme: Boolean): SubmitVatReturnModel = SubmitVatReturnModel(
+    1, 2, 3, 4, 5, 6, 7, 8, 9, flatRateScheme, dateToUse, dateToUse, dateToUse
```

Comment:
When you've updated the `amount` to be formatted with £ and commas, could you update this test data to be larger values with decimals etc please